wremnants/CardTool.py

```diff
@@ -954,9 +973,10 @@ def writeHistWithCharges(self, h, name):
     def writeHist(self, h, proc, syst, setZeroStatUnc=False, decorrByBin={}, hnomi=None):
         if self.skipHist:
             return
-        if self.project:
-            axes = self.project[:]
-            if "charge" in h.axes.name and "charge" not in axes and not self.xnorm :
+        if self.fit_axes:
+            axes = self.fit_axes[:]
+            # TODO: SHould probably not auto set this anymore
+            if self.charge_ax in h.axes.name and not self.xnorm and self.charge_ax not in axes:
```

Comment:
This is still a remnant where "charge" is magically added if it is not there already, I agree with the comment and suggest to remove this line and the one below

Reply:
Does it still need to be added explicitly in setFitAxes ? Should I remove that too?